packages/website/gatsby-node.js

```diff
@@ -49,14 +50,21 @@ exports.createPages = async ({ graphql, actions }) => {
     fromPath: `/`,
     isPermanent: true,
     redirectInBrowser: true,
-    toPath: `/docs`,
+    toPath: defaultPath,
   });
 
   createRedirect({
     fromPath: `/docs`,
     isPermanent: true,
     redirectInBrowser: true,
-    toPath: `/docs/getting-started`,
+    toPath: defaultPath,
+  });
+
+  createRedirect({
+    fromPath: `/docs/`,
```

Comment:
So even with the netlify plugin, we need to create redirects from both `/docs` and `/docs/`?

Reply:
It seems so. I will do another test later just to check again.

Reply:
Ok, I'll wait for your report 🤙

Reply:
Tested just now and the later redirect is not needed. [Netlify treats redirect rules](https://www.netlify.com/docs/redirects/#trailing-slash) always without trailing slash, so these two rules behave like the first one in the end:

`/docs` -> `/docs/getting-started`
`/docs/` -> `/docs/getting-started`

Reply:
Even better 🙂